Editors/README.md

```diff
@@ -143,6 +143,54 @@ As a test, open a Swift file, put the cursor on top of a symbol in normal mode a
 :call CocAction('jumpDefinition')
 ```
 
+### Neovim 0.8 and above
+since version 0.8, neovim has native LSP support, which can be used to connect to sourcekit-lsp directly. To do so, add the following to 
+a .lua config file (such as `lua/swift.lua`):
+
+```lua
+require'lspconfig'.sourcekit.setup{
+        cmd = {'$TOOLCHAIN_PATH/usr/bin/sourcekit-lsp'}
```

Comment:
One more to adjust the indentation on. :)

Reply:
oops! my bad! fixed :)